src/control/settings/Settings.cpp

```diff
@@ -374,7 +374,7 @@ void Settings::parseItem(xmlDocPtr doc, xmlNodePtr cur)
 	}
 	else if (xmlStrcmp(name, (const xmlChar*) "sidebarWidth") == 0)
 	{
-		this->sidebarWidth = g_ascii_strtoll((const char*) value, NULL, 10);
+		this->sidebarWidth = std::max(g_ascii_strtoll((const char*) value, NULL, 10), 50L);
```

Comment:
you forgot to include the math header :wink:

Reply:
the header was somehow included already, over other headerfiles